.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{

```

Comment:
we should update the gitignore: .vs sounds like vscode settings, which shouldn't be uploaded to git